### PR TITLE
Fixed defending aura bug in pre-renewal 

### PR DESCRIPTION
**Pull request by milky-milk**
In pre-re Defender doesn't reduce damage from Acid Demonstration
http://hercules.ws/board/tracker/issue-8560-pre-renewal-defending-aura-bug/

### CODE REVIEW

File: src/map/battle.c [hunk 1]
```diff
@@ -3001,8 +3001,14 @@ int64 battle_calc_damage(struct block_list *src,struct block_list *bl,struct Dam
 #endif
 
 		if(sc->data[SC_DEFENDER] &&
+#ifdef RENEWAL
 			((flag&(BF_LONG|BF_WEAPON)) == (BF_LONG|BF_WEAPON) || skill_id == CR_ACIDDEMONSTRATION))
 			damage = damage * ( 100 - sc->data[SC_DEFENDER]->val2 ) / 100;
+#else
+			(flag&(BF_LONG|BF_WEAPON)) == (BF_LONG|BF_WEAPON)) // In pre-re Defender doesn't reduce damage from Acid Demonstration
+			damage = damage * ( 100 - sc->data[SC_DEFENDER]->val2 ) / 100;
```
Comment (dastgir) [inline src/map/battle.c]: why not make this common, and only include
condition part in #ifdef ?

**Comment by kyeme** [comment]
Up~

**Comment by MishimaHaruna** [comment]
@milky-milk Could you edit this as suggested by @dastgir (by amending the commit, rather than making a new one) ? The line `damage = damage * ( 100 - sc->data[SC_DEFENDER]->val2 ) / 100;` doesn't need to be duplicated.

Otherwise I will edit it while merging

**Comment by milky-milk** [comment]
Hello! 
Sorry for delay, I will fix it soon.

**Comment by MishimaHaruna** [comment]
Thank you for your contribution!

**Pull request merged**